Dockerfile

```diff
@@ -1,145 +1,166 @@
 #--------------------------------------
 # Target image to build
 #--------------------------------------
-ARG IMAGE=latest
+ARG TARGET=latest
 
 #--------------------------------------
 # Ubuntu base image to use
 #--------------------------------------
-ARG FLAVOR=latest
+ARG FLAVOR=bionic
```

Comment:
Is `FLAVOR` the best term we can use?

Reply:
It's just a term i selected for building the different ubuntu versions.

Reply:
As my intention is that outsiders will start using these files, I'd like to align it with mainstream terminology. Although.. I would still prefer to support just one flavor at a time :)